### DISCUSSION TITLE
Question regarding exogenous and endogenous stablecoins

### DISCUSSION THREAD

**Discussion by LordAdhaar** [Cyfrin/foundry-full-course-cu#2901] [General]
Exogenous stablecoins like USDT/USDC are pegged to the dollar.
The dollar is also technically a stablecoin which was backed by the gold standard.
The gold standard was removed making the dollar now endogenous.
So exogenous stablecoins are also technically endogenous? Since they are backed by USD and USD is backed by nothing but itself?

**Comment by EngrPips** [comment]
Plot twist lol.

**Comment by EngrPips** [comment]
But then let's assume USD is still backed by Gold, so what is Gold backed by? @LordAdhaar

**Reply by LordAdhaar** [reply]
Gold is backed by the value it provides. It is used for jewellery, electronics, etc. 

But USD has value becuase we believe it does. But again that is my dubious speculation, I could be totally wrong. What do you think @EngrPips ?

**Reply by EngrPips** [reply]
I think it would be better to say Gold is backed by its limited supply and the fact that you can't make Gold out of thin air like we can print money. But at a point, there will be something that is not backed by anything but its intrinsic value. So if we agree that the intrinsic value of USD is sufficient to back a Stablecoin then we have a Stablecoin at least in essence until USD proves otherwise.

**Reply by LordAdhaar** [reply]
Interesting. 

There is coin called XAUT (Tether Gold). It is a stable coin backed by gold.

**Comment by jackmill3** [comment]
A well-chosen [mens necklaces & pendants](https://illiciumlondon.co.uk/mens-necklaces/) can add a touch of sophistication and personality to any outfit. From minimalist chains to bold statement pieces, these accessories allow men to express style, enhance their look, and make a subtle yet impactful fashion statement.

**Comment by JackieJ35** [comment]
This [px-to-em converter](https://pixelsconverters.com/px-to-em/) is actually very useful, especially when working on responsive typography. It saves a lot of time compared to doing manual calculations and helps keep font sizing consistent across breakpoints. Thanks for sharing such a practical tool.
